plugins/services/src/js/containers/services/__tests__/ServicesTableRenderers-test.js

```diff
@@ -39,27 +29,27 @@ describe("ServicesTable", function() {
 
   describe("#renderStats", function() {
     it("renders resources/stats cpus property", function() {
-      var cpusCell = cpuRenderer(healthyService);
+      var cpusCell = renderer.create(cpuRenderer(healthyService)).toJSON();
 
-      expect(cpusCell).toEqual(renderNumberCell(1));
+      expect(cpusCell).toMatchSnapshot();
```

Comment:
Components created with `React.memo` show up as `<UNDEFINED />` (see https://github.com/facebook/jest/issues/7883) so I switched these to regular snapshot tests.